<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnOffWizardPage</class>
 <widget class="QWizardPage" name="OnOffWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>On/Off Control</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="allOff">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>All Off</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="allOn">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>All On</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="off0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 0 Off</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="on0">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 0 On</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="off1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 1 Off</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="on1">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 1 On</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="off2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 2 Off</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="on2">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 2 On</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="off3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 3 Off</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb_off.png</normaloff>:/icons/lightbulb_off.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="on3">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Port 3 On</string>
     </property>
     <property name="icon">
      <iconset resource="../rc/icons.qrc">
       <normaloff>:/icons/lightbulb.png</normaloff>:/icons/lightbulb.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
